package.json

```diff
@@ -92,6 +92,7 @@
     "http-shutdown":"~1.2.2",
     "inquirer": "~8.2.5",
     "lru-cache": "~10.0.1",
+    "md5": "~2.3.0",
```

Comment:
I don't think it's needed to add the md5 lib. Check if hash lib works as we already have it included.
example to search for on the code: "const messageHash = hash(msg);"

Reply:
good idea :) tested working correct